library/templates/dcp_cpdb.yml

```diff
@@ -0,0 +1,38 @@
+dataset:
+  name: &name dcp_cpdb
+  version: "{{ version }}"
+  acl: private 
+  
+  source:
+    url:
+      path: library/tmp/cpdb/cpdb_dcpattributes_poly_18.zip
```

Comment:
I think we need to specify the subpath for the shapefiles + also, the path should be for the whole zip file of all the geometries